docs/getting-started/quick-start.md

```diff
@@ -2,14 +2,185 @@
 
-_Here we provided a small sample on OpenShift_
\ No newline at end of file
+This topic provides the following information on how to deploy application and connect it to a a backing service:
+
+1. [Prerequisites](#prerequisites)
+2. [Creating a database instance](#creating-a-database-instance)
+3. [Deploying an application](#deploying-an-application)
+4. [Connecting the application to a backing service](#connecting-the-application-to-a-backing-service)
+
+## Prerequisites
+
+The following components must be installed and configured:
+
+- Kubernetes cluster (**Note:** You can use [minikube](https://minikube.sigs.k8s.io/) or
+  [kind](https://kind.sigs.k8s.io/), locally)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Service Binding Operator](installing-service-binding).
+
+## Creating a database instance
+
+The application is going to connect to a PostgreSQL database backend.  The
+PostgreSQL can be setup using the [Crunchy PostgreSQL operator from
+OperatorHub.io][crunchy].
+
+The installation of the operator doesn't create a database instance for
+connection.
+
+1. To create a database instance, you need to create custom resource
+(`Pgcluster`) and that will trigger the operator reconciliation.  For
+convenience, run this command to create `Pgcluster` custom resource:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/pgcluster.yaml
+```
+
+2. Ensure all the pods in `my-postgresql` is running (it will take few minutes):
+
+```bash
+kubectl get pod -n my-postgresql
+```
+
+You should see output something like this:
+
+```
+NAME                                          READY   STATUS    RESTARTS   AGE
+backrest-backup-hippo-9gtqf                   1/1     Running   0          13s
+hippo-597dd64d66-4ztww                        1/1     Running   0          3m33s
+hippo-backrest-shared-repo-66ddc6cf77-sjgqp   1/1     Running   0          4m27s
+```
+
+The database will be empty by default, it requires the schema and sample data to
+work with the application.
+
+3. You can initialize the database with the schema and sample data using this
+command:
+
+```bash
+bash <(curl -s https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/init-database.sh)>
+```
+
+Now the database is ready to connect from application.  The next section
+explains how to configure application:
+
+## Deploying an application
+
+1. Deploy the `spring-petclinic-rest` app with this `Deployment` configuration:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/app-deployment.yaml
+```
+
+2. Port forward the application port and try to access it from your local system
+
+```
+kubectl port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n my-postgresql
```

Comment:
wouldn't it be better to setup an ingress?

Reply:
cc @pedjak

Reply:
For those who are trying this thing on minikube, it requires more work: https://minikube.sigs.k8s.io/docs/handbook/addons/ingress-dns/